front/cypress/integration/content_builder/content_builder_navigation.ts

```diff
@@ -0,0 +1,76 @@
+import { randomString } from '../../support/commands';
+
+describe('Content builder navigation', () => {
+  let projectId = '';
+  let projectSlug = '';
+
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.getAuthUser().then((user) => {
+      const projectTitle = randomString();
+      const projectDescriptionPreview = randomString();
+      const projectDescription = 'Original project description.';
+      const userId = user.body.data.id;
+
+      cy.apiCreateProject({
+        type: 'continuous',
+        title: projectTitle,
+        descriptionPreview: projectDescriptionPreview,
+        description: projectDescription,
+        publicationStatus: 'published',
+        participationMethod: 'ideation',
+        assigneeId: userId,
+      }).then((project) => {
+        projectId = project.body.data.id;
+        projectSlug = projectTitle;
+        cy.visit(`/projects/${projectSlug}`);
```

Comment:
Do you need this? You navigate away from it in your first test.

Reply:
True - I'll remove it. Thanks!